tests/Gedmo/Mapping/Fixture/Document/EmbeddedRevisionable.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Doctrine Behavioral Extensions package.
+ * (c) Gediminas Morkevicius <gediminas.morkevicius@gmail.com> http://www.gediminasm.org
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Gedmo\Tests\Mapping\Fixture\Document;
+
+use Doctrine\ODM\MongoDB\Mapping\Annotations as ODM;
+use Doctrine\ODM\MongoDB\Types\Type;
+use Gedmo\Mapping\Annotation as Gedmo;
+
+/**
+ * @author Fabian Sabau <fabian.sabau@socialbit.de>
+ *
+ * @ODM\EmbeddedDocument
+ */
+#[ODM\EmbeddedDocument]
+class EmbeddedRevisionable
```

Comment:
```suggestion
final class EmbeddedRevisionable
```

Reply:
The MongoDB ODM doesn't have an issue with final embedded objects?  (I know there are quirks with final in some places in the OMs but I don't remember what the deal is with embeddable)

Reply:
I'm not sure, but I don't have enough time either.
So for now, let's keep it extensible.